src/FakeItEasy/Configuration/FakeConfigurationManager.cs

```diff
@@ -100,12 +102,14 @@ private static Expression BuildArgumentThatMatchesAnything<TValue>()
 
         private static FakeManager GetFakeManagerCallIsMadeOn(ParsedCallExpression parsedCallExpression)
         {
-            if (parsedCallExpression.CallTarget == null)
-            {
-                throw new ArgumentException("The specified call is not made on a fake object.");
-            }
+            return parsedCallExpression.CallTarget == null
+                ? null

```

Comment:
> I'm wondering if we shouldn't keep all the code as is, except to explicitly check the manager for nullness before we add a rule to it.

Sure, that would work.

I have another question though. When `parsedCallExpression.CallTarget == null` we are returning `null` from `GetFakeManagerCallIsMadeOn` which means no exception is thrown from `EnsureCallIsMadeOnAFake`. That implies that `parsedCallExpression.CallTarget == null` means "the calls are made on a fake". Is that correct?

More generally, all calls to `GetFakeManagerCallIsMadeOn` will now receive a null instead of an exception being thrown when `parsedCallExpression.CallTarget == null`. Is that what we want?